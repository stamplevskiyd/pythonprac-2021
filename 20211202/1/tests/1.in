class C(metaclass=dump):
    def __init__(self, val):
        self.val = val

    def add(self, other, another=None):
        return self.val + other + (another or self.val)

    def __str__(self):
        return("Fake str convertion")

c = C(10)
print(c.add(9))
print(c)
