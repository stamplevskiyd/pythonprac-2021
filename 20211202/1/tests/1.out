__init__: (10,), {}
add: (9,), {}
29
__str__: (), {}
Fake str convertion
